test/functional/fixtures/runner/uncaught-js-errors/test_config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "baseUrl": "http://localhost:3000/runner/uncaught-js-errors/"

```

Comment:
`http://localhost:3000` is not necessary because we have the `test_config.json` file with this prefix in the `/fixtures` path